packages/ui/src/components/Button/index.tsx

```diff
@@ -1,41 +1,104 @@
+import { I18nKey } from '@logto/phrases-ui';
 import classNames from 'classnames';
-import { ReactNode } from 'react';
+import { HTMLProps, ReactElement, ReactNode, useEffect, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
 
+import DangerousRaw from '../../../../console/src/components/DangerousRaw';
+import { Ring as Spinner } from '../../../../console/src/components/Spinner';
```

Comment:
Should not reference any component from the console package here, as they are two different packages under the mono repo.

Reply:
yeah, I see the console, the project is not used anywhere. So I hesitated to import it into the UI project. I think it's better to keep the button and spinner components in a shared project.